feat: Unified Order History Receipt URL

grmartin opened the pull request:
SONIC-279

grmartin committed:
feat: Unified Order History Receipt URL

SONIC-279

shafqatfarhan commented on .env.development:
```diff
@@ -18,6 +18,7 @@ SITE_NAME=localhost
 MARKETING_SITE_BASE_URL=http://localhost:18000
 SUPPORT_URL=http://localhost:18000/support
 ORDER_HISTORY_URL=http://localhost:8140/orders/unified/order_history/
+RECEIPT_URL=http://localhost:8140/orders/unified/order_receipt
```
```suggestion
RECEIPT_URL=http://localhost:8140/orders/unified/order_receipt/
```

grmartin replied:
I still am chaffing at this... whats the logic?

grmartin replied:
Also that is here: https://github.com/edx/edx-internal/pull/10177

shafqatfarhan reviewed (approved):
Approved. Don't forget to add `RECEIPT_URL` in edx-internal for stage and prod envs.

grmartin committed:
fix: Update .env.development

trailing slash is what nginx appends to URLs during routing to the appropriate service. If it is not present, no endpoint would match

Co-authored-by: Shafqat Farhan <shafqat.farhan@arbisoft.com>
